config/manifests/osl/bases/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: logic-operator-rhel8.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KogitoServerlessBuild is the Schema for the kogitoserverlessbuilds
+        API
+      displayName: Kogito Serverless Build
+      kind: KogitoServerlessBuild
+      name: kogitoserverlessbuilds.sw.kogito.kie.org
+      version: v1alpha08
+    - description: KogitoServerlessPlatform is the Schema for the kogitoserverlessplatforms
+        API
+      displayName: Kogito Serverless Platform
+      kind: KogitoServerlessPlatform
+      name: kogitoserverlessplatforms.sw.kogito.kie.org
+      version: v1alpha08
+    - description: KogitoServerlessWorkflow is the Schema for the kogitoserverlessworkflows
+        API
+      displayName: Kogito Serverless Workflow
+      kind: KogitoServerlessWorkflow
+      name: kogitoserverlessworkflows.sw.kogito.kie.org
+      version: v1alpha08
+  description: OpenShift Serverless Logic Operator
+  displayName: openshift-serverless-logic-operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - kogito
+  - serverless
+  - serverlessworkflow
+  links:
+  - name: OpenShift Serverless Logic Operator
+    url: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/index.html
+  maturity: alpha
+  provider:
+    name: logic-operator-rhel8
+  relatedImages:
+  - image: registry.redhat.io/openshift-serverless-1-tech-preview/logic-operator-rhel8@sha256:aae0198cbd4a9d92130437d3869b6da8854ba3a7c229956e172b621aac3261f3
```

Comment:
You explained to me that this is just for the upgrade. In that case, can't we use the tag `1.29.0` instead of the `sha256`? Or the `sha256` is really needed?

I'm just thinking that it would be easier if we don't have to update this field with the sha of the Operator image and just use the tag.

Reply:
IMHO for now we should stick with the tag.
For the sha256, it is better to let osbs handle it.